presto-cache/src/main/java/com/facebook/presto/cache/CacheConfig.java

```diff
@@ -15,15 +15,22 @@
 
 import com.facebook.airlift.configuration.Config;
 import com.facebook.airlift.configuration.ConfigDescription;
+import com.facebook.presto.hive.CacheQuotaScope;
+import io.airlift.units.DataSize;
 
 import java.net.URI;
+import java.util.Optional;
+
+import static com.facebook.presto.hive.CacheQuotaScope.GLOBAL;
 
 public class CacheConfig
 {
     private boolean cachingEnabled;
     private CacheType cacheType;
     private URI baseDirectory;
     private boolean validationEnabled;
+    private CacheQuotaScope cacheQuotaScope = GLOBAL;
+    private Optional<DataSize> defaultCacheQuota = Optional.empty();
```

Comment:
Why it is called default? Isn't it the cacheQuota allocated for this config, it could be default or the one configured?

Reply:
I just updated the project doc link and explained it there